### PR TITLE
installation: fix invenio_base imports

### PR DESCRIPTION
**Pull request by SamiHiltunen**
* FIX Adds missing `invenio_base` dependency and amends past
  upgrade recipes following its separation into standalone package.

Signed-off-by: Sami Hiltunen <sami.mikael.hiltunen@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
- [ ] update copyright year (`, 2015`) in https://github.com/inveniosoftware/invenio-webhooks/pull/2/files#diff-cb6eebbaedea633f8b1d434e39ed0852R4

**Comment by SamiHiltunen** [comment]
Copyright updated.

File: requirements-devel.txt [hunk 1]
```diff
@@ -27,3 +27,5 @@
 #
 #     -e git+git://github.com/mitsuhiko/werkzeug.git#egg=Werkzeug
```
Comment (jirikuncar) [inline requirements-devel.txt]: please remove this TODO comment and add invenio-oauth2server. thx

**Comment by SamiHiltunen** [comment]
@jirikuncar TODO removed + added oauth2server

**Pull request merged**